backend/WebApi/Controllers/ModificationEntryController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using AutoMapper;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using ModelLayer.DataTransferObjects;
+using ModelLayer.Models;
+using NSwag.Annotations;
+using ServiceLayer;
+
+namespace WebApi.Controllers
+{
+    [Route("api/home")]
+    [ApiController]
+    public class ModificationEntryController : ControllerBase
+    {
+        private readonly IMapper _mapper;
+        private readonly IModificationEntryService modificationEntryService;
+
+        public ModificationEntryController(IMapper mapper, IModificationEntryService modificationEntryService)
+        {
+            this._mapper = mapper;
+            this.modificationEntryService = modificationEntryService;
+        }
+
+        [HttpGet]
+        [SwaggerResponse(HttpStatusCode.OK, typeof(List<ModificationEntryDto>), Description = "successfully found")]
+        [SwaggerResponse(HttpStatusCode.NotFound, typeof(void), Description = "contact not found")]
+        public async Task<IActionResult> GetSortedListByType(MODEL_TYPE modelDataType)
```

Comment:
Endpunkt ist ungeschützt und kann ohne AccessToken abgefragt werden

Reply:
und wie mache ich das bitte? bzw wo finde ich denn überhaupt diesen schutz?

Reply:
https://docs.microsoft.com/de-de/aspnet/web-api/overview/security/authentication-and-authorization-in-aspnet-web-api

Reply:
ok